rllib/offline/estimators/direct_method.py

```diff
@@ -20,23 +20,27 @@
 logger = logging.getLogger()
 
 
-@ExperimentalAPI
+@DeveloperAPI
 def train_test_split(
```

Comment:
so where does this get invoked, now that we will require train and test datasets to be specified ahead of time?

Reply:
The whole function gets deleted in the next PR :) the user should split them ahead of time, or we can make this a utility in Ray Train/Datasets.